server/plugin.go

```diff
@@ -606,7 +606,12 @@ func (p *Plugin) GetToDo(ctx context.Context, user *gitlab.UserInfo) (bool, stri
 				continue
 			}
 			notificationCount++
-			notificationContent += fmt.Sprintf("* %v : [%v](%v)\n", n.ActionName, n.Target.Title, n.TargetURL)
+
+			if n.ActionName == ActionNameMemberAccessRequest {
```

Comment:
Need your thoughts on this I think we switch case here as In the future we can have different ActionName which need to be handled separately.